positions.go

```diff
@@ -14,7 +16,35 @@ type Position struct {
 
 // GetPositions returns all the positions associated with an account.
 func (c Client) GetPositions(a Account) ([]Position, error) {
+	return c.GetPositionsParams(a, PositionParams{})
+}
+
+// PositionParams encapsulates parameters known to the RobinHood positions API
+// endpoint.
+type PositionParams struct {
+	NonZero bool
+}
+
+// Encode returns the query string associated with the requested parameters
+func (p PositionParams) Encode() string {
```

Comment:
I don't think this needs to be an exported method?
```suggestion
func (p PositionParams) encode() string {
```